src/ui.data-table/configs/default.config.js

```diff
@@ -1,6 +1,6 @@
 export default /*tw*/ {
   wrapper: "relative",
-  headerCellGeneral: "p-4 text-sm font-normal text-gray-500 text-left",
+  headerCellGeneral: "!overflow-visible p-4 first:p-5 text-sm font-normal text-gray-500 text-left",
```

Comment:
Why do we need `!overflow-visible`, it goes by default

Reply:
Btw, maybe it's better to rename `headerCellGeneral` to `headerCellBase`

Reply:
okay